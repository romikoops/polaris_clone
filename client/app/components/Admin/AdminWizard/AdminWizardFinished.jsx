import React, { Component } from 'react'
import PropTypes from '../../../prop-types'
import styles from '../Admin.scss'
// import { AdminHubTile } from '../';
// import { RoundButton } from '../../RoundButton/RoundButton';
// import {v4} from 'node-uuid';
// import Select from 'react-select';
// import '../../../styles/select-css-custom.css';
// import styled from 'styled-components';
// import FileUploader from '../../../components/FileUploader/FileUploader';
export class AdminWizardFinished extends Component {
  constructor (props) {
    super(props)
    this.nextStep = this.nextStep.bind(this)
  }
  nextStep () {
    this.props.adminTools.goTo('/admin/wizard/finished')
  }

  render () {
    const { theme } = this.props
    const textStyle = {
      background:
        theme && theme.colors
          ? `-webkit-linear-gradient(left, ${theme.colors.primary},${theme.colors.secondary})`
          : 'black'
    }

    return (
      <div className="flex-100 layout-row layout-wrap layout-align-start-center">
        <div className="flex-100 layout-row layout-wrap layout-align-start-start">
          <div className={`flex-100 layout-row layout-align-start-center ${styles.sec_title}`}>
            <h1 className={` ${styles.sec_title_text} flex-none`} style={textStyle}>
              Finished!
            </h1>
          </div>

          <div className={`flex-100 layout-row layout-align-start-center ${styles.sec_title}`}>
            <p className={` ${styles.sec_title_text} flex-none`} style={textStyle}>
              Browse through the links on the left to see your data!
            </p>
          </div>
        </div>
      </div>
    )
  }
}
AdminWizardFinished.propTypes = {
  theme: PropTypes.theme,
  adminTools: PropTypes.shape({
    goTo: PropTypes.func
  }).isRequired
}

AdminWizardFinished.defaultProps = {
  theme: null
}

export default AdminWizardFinished
