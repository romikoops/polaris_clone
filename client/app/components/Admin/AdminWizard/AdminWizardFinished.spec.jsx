import * as React from 'react'
import { shallow } from 'enzyme'
import { theme, identity } from '../../../mock'
import AdminWizardFinished from './AdminWizardFinished'

const propsBase = {
  theme,
  adminTools: {
    goTo: identity
  }
}

test('shallow render', () => {
  expect(shallow(<AdminWizardFinished {...propsBase} />)).toMatchSnapshot()
})
