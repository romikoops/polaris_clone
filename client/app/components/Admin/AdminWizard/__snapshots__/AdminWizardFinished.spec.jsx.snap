// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow render 1`] = `
<div
  className="flex-100 layout-row layout-wrap layout-align-start-center"
>
  <div
    className="flex-100 layout-row layout-wrap layout-align-start-start"
  >
    <div
      className="flex-100 layout-row layout-align-start-center sec_title"
    >
      <h1
        className=" sec_title_text flex-none"
        style={
          Object {
            "background": "-webkit-linear-gradient(left, #333,#fafafa)",
          }
        }
      >
        Finished!
      </h1>
    </div>
    <div
      className="flex-100 layout-row layout-align-start-center sec_title"
    >
      <p
        className=" sec_title_text flex-none"
        style={
          Object {
            "background": "-webkit-linear-gradient(left, #333,#fafafa)",
          }
        }
      >
        Browse through the links on the left to see your data!
      </p>
    </div>
  </div>
</div>
`;
